### DISCUSSION TITLE
Old data fetch on website

### DISCUSSION THREAD

**Discussion by michaelstringer** [weather-gov/api#503] [Q&A]
Hi there,

We're still getting stale data on our website, though a quick cross-check on Postman reveals current data. I wish I could figure out once and for all why this happens. Our API call is from https://api.weather.gov/gridpoints/MFR/107,60/forecast

Thanks in advance for any help!

**Comment by JJKraw** [comment]
See this discussion if you haven't read it:

  https://github.com/weather-gov/api/discussions/329

I haven't seen (or else I missed) any more recent updates about whether the team has finally been given the go-ahead to do the non-trivial work to get this corrected.

My work-around has been to cache my results and then ignore further queries that are older than what's in my cache. It has worked pretty well (meaning that at worst, a few times a day I'm getting current data).

**Comment by efishinsea** [comment]
I'm having the same issue. 

Oddly, if I stop sending the headers in the curl request (CURLOPT_HTTPHEADER) it works ok.  But with the header array of "Accept: application/json" it fails and gives me last week's data.

Seems to be many threads about this around here.
